doc/how_to/migrate/from_streamlit.md

```diff
@@ -0,0 +1,950 @@
+[Templates Section](https://panel.holoviz.org/reference/index.html#templates) of the
+[Component Gallery](https://panel.holoviz.org/reference/index.html).
+
+You use `pn.panel` similarly to `st.write` to *magically* display
+Python objects like (markdown) strings, dataframes, plots and more. Check out the
+[`pn.panel` API Reference](https://panel.holoviz.org/api/panel.pane.html#panel.pane.panel) for the
+details.
+
+You use `.servable` to specify the Panel objects you want to add the Panel *template* when
+served as a web app.
+
+You *serve* and *show* the app with *autoreload* via
+
+```bash
+panel serve app.py --autoreload --show
+```
+
+See the [Command Line Guide](https://panel.holoviz.org/how_to/server/commandline.html) for more
+command line options.
+
+The application looks like
+
+![Panel Hello World Example](https://user-images.githubusercontent.com/42288570/243343688-e6212faa-7b05-4686-b113-b67cd21e1063.png)
+
+### Basic Migration Steps
+
+You should replace
+
+- `import streamlit as st` with `import panel as pn` and
+- `st.write` with `pn.panel`.
+
+You will also have to
+
+- add `pn.extension` to configure your Panel application via arguments like `sizing_mode` and `template`.
+- add `.servable` to the Panel objects you want to include in your apps *template* when served as
+a web app.
+
+For production you will also have to
+
+- Migrate some of your app configuration to `panel serve` command line options or environment
+variables.
+
+You won't have to
+
+- Provide your email or
+- opt out of telemetry data collection.
+
+We have never collected or had plans to collect telemetry data from our users apps.
+
+## Outputs
+
+In Panel the objects that can display your Python objects are called *panes*. With Panels *panes*
+you will be able to
+
+- get notifications about interactions like click events on your plots and tables and react to them.
+- use unique data visualization ecosystems like HoloViz, ipywidgets and VTK.
+
+Check out the
+[Panes Section](https://panel.holoviz.org/reference/index.html#panes) of
+the [Component Gallery](https://panel.holoviz.org/reference/index.html) for the full list of
+*panes*.
+
+### Streamlit Matplotlib Example
+
+```python
+import numpy as np
+import streamlit as st
+
+import matplotlib.pyplot as plt
+
+data = np.random.normal(1, 1, size=100)
+fig, ax = plt.subplots(figsize=(8,4))
+ax.hist(data, bins=20)
+
+st.pyplot(fig)
+```
+
+The app looks like
+
+![Streamlit Matplotlib Example](https://user-images.githubusercontent.com/42288570/243348167-4d2a3c36-59dc-4df5-b266-84dd1837f2b6.png)
+
+### Panel Matplotlib Example
+
+You will find Panels output *panes* in `pn.pane` module.
+
+```python
+import panel as pn
+import numpy as np
+
+from matplotlib.figure import Figure
+
+pn.extension(sizing_mode="stretch_width", template="bootstrap")
+
+data = np.random.normal(1, 1, size=100)
+fig = Figure(figsize=(8,4))
+ax = fig.subplots()
+ax.hist(data, bins=20)
```

Comment:
To keep it consistent, unless there's an issue doing it this way?
```suggestion
data = np.random.normal(1, 1, size=100)
fig, ax = plt.subplots(figsize=(8,4))
ax.hist(data, bins=20)
```

Reply:
Oh I just saw the note below. Maybe the note can be moved above.

Reply:
I've moved it above